src/Umbraco.Core/Composing/ComponentComposer.cs

```diff
@@ -9,7 +11,7 @@ public abstract class ComponentComposer<TComponent> : IComposer
         where TComponent : IComponent
     {
         /// <inheritdoc />
-        public virtual void Compose(Composition composition)
+        public virtual void Compose(IUmbracoBuilder composition)
```

Comment:
I think it's wise to update the parameter names when replacing the types.

Reply:
I thought I got all of them, I missed 2

Reply:
resolved 73bd42a